dist/extras/amd.js

```diff
@@ -55,7 +55,9 @@
       return {
         setters: setters,
         execute: function () {
-          module.exports = amdExec.apply(exports, depModules) || module.exports;
+          const amdResult = amdExec.apply(exports, depModules);
+          if (amdResult !== null && amdResult !== undefined)
```

Comment:
Why testing both null and undefined? Undefined is not enough ?
I'm just passing by and wondering don't feel forced to answer :)

Reply:
Agreed, I've changed it to just check for `undefined`, unless someone tells me that `null` is a possibility (eg: deletion edge case?).